images/linux/scripts/installers/chromium-browser.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+################################################################################
+##  File:  chromium-browser.sh
+##  Desc:  Installs Chromium browser
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/os.sh
+
+if ! isUbuntu20 ; then
```

Comment:
I believe it can be removed, because we invoke script only ubuntu 20.04 template.